Source/Scene/Expression.js

```diff
@@ -346,6 +346,12 @@ define([
             //>>includeEnd('debug');
             val = createRuntimeAst(expression, args[0]);
             return new Node(ExpressionNodeType.UNARY, call, val);
+        } else if (call === 'cos') {
+            if (args.length < 1 || args.length > 1) {
+                throw new DeveloperError('Error: ' + call + ' requires exactly one argument.');
+            }
```

Comment:
Wrap the `DeveloperError` in a pragma so that it is not included in release builds. Look at the `call === 'abs'` block right above.